presto-spi/src/main/java/com/facebook/presto/spi/type/VarcharType.java

```diff
@@ -28,6 +28,7 @@
 {
     public static final int MAX_LENGTH = Integer.MAX_VALUE;
     public static final VarcharType VARCHAR = new VarcharType(MAX_LENGTH);
+    public static final String VARCHAR_UNBOUNDED = "varchar(2147483647)";
```

Comment:
This doesn't look right. As we discussed in the past, varchar(x) and unbounded varchar are different types. We had concluded we would internally represent unbounded varchar as varchar(max) and disallow explicit uses of varchar(max) to ensure the types don't alias to each other. Maybe that can't be done? Let me think further about the implications.